cypress/integration/examples/homepage.spec.js

```diff
@@ -0,0 +1,129 @@
+
+context('HOME PAGE', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000')
```

Comment:
These (or similar files) will need to be stubbed so that we can run the tests while the API is live.